protocol/errors.json

```diff
@@ -182,6 +182,39 @@
   "91004": "unable to automatically re-enter presence channel",
   "91005": "presence state is out of sync",
   "91100": "member implicitly left presence channel (connection closed)",
+  "92000": "Invalid MapOp: key required",
+  "92001": "Channel state max objects limit exceeded",
+  "92002": "Channel message must contain state operation message",
+  "92003": "Invalid state message object ID length",
+  "92004": "Invalid state message object ID not a valid base64 encoded string",
+  "92005": "Invalid state message object ID 'root' cannot be used in action",
+  "92006": "Invalid state message must contain either an object or operation",
+  "92007": "Invalid state message cannot contain both an operation and object",
+  "92008": "Invalid state message object ID does not match expected value",
+  "92009": "Invalid state message object ID required",
+  "92010": "Invalid state message object ID nonce required",
+  "92011": "Invalid state message unknown operation",
+  "92012": "Invalid state message object ID required",
+  "92013": "State messages do not support form encoding",
+  "92014": "State message required",
+  "92015": "State operation required",
+  "92016": "Invalid operation: expected a create operation",
+  "92017": "Invalid map operation: MapOp should not be provided",
+  "92018": "Invalid map operation: MapOp should be provided",
+  "92019": "Invalid map operation: Data required",
+  "92020": "Invalid map operation: unknown action",
+  "92021": "Invalid counter operation: CounterOp should be provided",
+  "92022": "Invalid counter operation: CounterOp should have non zero value:",
+  "92023": "Invalid counter operation: Counter should not be provided",
+  "92024": "State data required",
+  "92025": "Invalid State Data: both data and object id can not be set",
+  "92026": "Invalid State Data: either data or object id must be set",
+  "92027": "State object required",
+  "92028": "Invalid state object: a map or counter required",
+  "92029": "State value required",
+  "92030": "Invalid counter operation: unknown action",
+  "92031": "Channel message required",
+  "92032": "Invalid channel message type: state type require",
```

Comment:
```suggestion
  "92032": "Invalid channel message type: state type required",
```

Reply:
also can't find a reference to this error message in the code

Reply:
It is used in the LiveObjectPool when checking the action type on the channel message